#request
uint8 payload_index #see enum class PayloadIndex in common_type.h
uint8 exposure_mode #see enum class ExposureMode in common_type.h
uint16 aperture      #see enum class Aperture in common_type.h
---
#response
bool result